### ISSUE TITLE
About the choice of momentum and eps for “batch normalization”.

### ISSUE THREAD

**Issue by qixuema** [octree-nn/ocnn-pytorch#26]
Hello, Professor Wang,
I noticed that in the code, the comments suggest using 0.1 and 1e-5 for certain parameters in PyTorch. However, it seems that in the actual usage, values like 0.01 and 0.001 are used. I was wondering about the reasoning behind this difference?

![image](https://github.com/octree-nn/ocnn-pytorch/assets/58554118/92730510-dd9c-4f44-be80-e7bfc654f4df)

https://github.com/octree-nn/ocnn-pytorch/blob/36072d8e8dc1ed803c77c34a3806a5c5dfb4af7a/ocnn/modules/modules.py#L17-L18

**Comment by wang-ps** [comment]
Originally, ocnn was implemented with TensorFlow and Caffe. When I tried to implement ocnn on Pytorch, I used the default values of batch norm according to my initial TensorFlow code for reference. 

According to my experience, the network is generally robust to these values. You can safely choose either setting as you desire. However, for the released trained weights, I used the parameter setting in Line 17.

**Comment by qixuema** [comment]
Cool! If the model proves to be robust with respect to these parameters, it would be excellent for the training process. Thank you very much for your assistance.
